fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/BestMvSelector.java

```diff
@@ -152,15 +177,38 @@ private void calculateStatistics(OptExpression expr, OptimizerContext context) {
         expr.setStatistics(expressionContext.getStatistics());
     }
 
+    private int calcSortScore(MaterializedView mv, Set<String> equivalenceColumns, Set<String> nonEquivalenceColumns) {
+        List<Column> schema = mv.getBaseSchema();
+        int score = 0;
+        for (Column col : schema) {
+            String columName = col.getName().toLowerCase();
+            if (equivalenceColumns.contains(columName)) {
+                score++;
+            } else if (nonEquivalenceColumns.contains(columName)) {
```

Comment:
it seems that simple range/in/eq predicates's column can be prefix of columns of rollup.

Reply:
yes, but range can be use in first column